### DISCUSSION TITLE
Combine 2 buttons for one action

### DISCUSSION THREAD

**Discussion by Scott216** [AntiMicroX/antimicrox#511] [Q&A]
I would like Page Up to be simulated, but only if both the left and right shoulder buttons are pressed together.  If only one is pressed, then nothing happens until the other is also pressed and held.  Is this possible with antimicrox?

**Comment by pktiuk** [comment]
This is not supported yet   
But there is already issue for this: https://github.com/AntiMicroX/antimicrox/issues/298
